config/build/rollup.config.js

```diff
@@ -34,74 +32,6 @@ const rootDir = ancesdir(__dirname);
  *               from the default behavior.
  */
 
-// Kahn's algorithm
-// https://en.wikipedia.org/wiki/Topological_sorting#Kahn%27s_algorithm
-const topoSort = (yarnWorkspacesOutput) => {
```

Comment:
Not needed.  WonderBlocks doesn't build in any dependency order, and after thinking about it a bit, we don't need to either. The build doesn't depend on build artifacts of other in-repo packages, so this simplifies our build a bunch!

Reply:
@kevinb-khan I tagged you mostly for these changes. I'm pretty sure we don't need to build in any dependency order.

Reply:
Does it still build if you reverse the topological sort? If it does, I think we're probably okay.

Reply:
Yep. I did a manual ordering in the reverse of what `topoSort()` produces and it built fine.